packages/web-app-files/src/views/shares/SharedViaLink.vue

```diff
@@ -8,10 +8,10 @@
           v-if="isEmpty"
           id="files-shared-via-link-empty"
           class="files-empty"
-          icon="link"
+          icon="file-forbid"
```

Comment:
Please revert to `link` icon for now. `file-forbid` is not a step forward... I agree that it needs improvement.